### PR TITLE
Successfully setup RSPEC

### PR DESCRIPTION
**Pull request by rupertlion**
## PT Link

https://www.pivotaltracker.com/story/show/160582694

## Installed RSPEC

* Added RSPEC to gem file
* Successfully ran RSPEC without tests

## What I learnt

* Patience!

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  #
+  # The different available types are documented in the features, such as in
+  # https://relishapp.com/rspec/rspec-rails/docs
+  config.infer_spec_type_from_file_location!
+
+  # Filter lines from Rails gems in backtraces.
+  config.filter_rails_from_backtrace!
+  # arbitrary gems may also be filtered via:
+  # config.filter_gems_from_backtrace("gem name")
+end
```
Comment (faraznaeem) [inline spec/rails_helper.rb]: Please remove all comments

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  # file, and it's useful to allow more verbose output when running an
+  # individual spec file.
+  if config.files_to_run.one?
+    # Use the documentation formatter for detailed output,
+    # unless a formatter has already been configured
+    # (e.g. via a command-line flag).
+    config.default_formatter = "doc"
+  end
+
+  # Print the 10 slowest examples and example groups at the
```
Comment (faraznaeem) [inline spec/spec_helper.rb]: same here

**Review by faraznaeem — CHANGES_REQUESTED** [review]

**Comment by faraznaeem** [comment]
Please provide a link to PT or waffle chore
and add a section with what you have learned

**Comment by rupertlion** [comment]
thumbs up emoji!

File: Gemfile [hunk 3]
```diff
@@ -20,6 +20,10 @@ group :development do
   gem 'pry-byebug'
   gem 'spring'
```
Comment (faraznaeem) [inline Gemfile]: you dont need pry-byebug, bacuse you are using pry rails

**Comment by rupertlion** [comment]
Updated - removed PRY byebug gem

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**